plugins/modules/podman_network.py

```diff
@@ -37,6 +38,11 @@
     description:
       - Driver to manage the network (default "bridge")
     type: str
+  force:
+    description:
+      - Remove all containers that use the network.
+        If the container is running, it is stopped and removed.
+    type: bool
```

Comment:
let's make it more visible and set as `False` by default, also in docs.
`default: False`